components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/impl/EndpointsApiServiceImpl.java

```diff
@@ -180,6 +193,16 @@ public Response endpointsEndpointIdPut(String endpointId, EndPointDTO body, Stri
             org.wso2.carbon.apimgt.rest.api.common.dto.ErrorDTO errorDTO = RestApiUtil.getErrorDTO(e.getErrorHandler());
```

Comment:
Import the org.wso2.carbon.apimgt.rest.api.common.dto.ErrorDTO and use the class name here.